src/AppServiceFileSystem.ts

```diff
@@ -58,4 +60,9 @@ export class AppServiceFileSystem extends AzExtTreeFileSystem<FileTreeItem> {
         this._etags.set(node.fullId, etag);
         await node.refresh();
     }
+
+    public appendLineToOutput(value: string, options?: { resourceName?: string, date?: Date }): void {
+        ext.outputChannel.appendLog(value, options);
+        ext.outputChannel.show(true);
```

Comment:
We [generally avoid](https://github.com/microsoft/vscode-azureappservice/issues/455) showing the output channel because it can get annoying